dask/dataframe/io/tests/test_csv.py

```diff
@@ -868,7 +869,7 @@ def test_to_csv_gzip():
     for npartitions in [1, 2]:
         a = dd.from_pandas(df, npartitions)
         with tmpfile('csv') as fn:
-            a.to_csv(fn, compression='gzip')
+            a.to_csv(fn, compression='gzip', sep=",")
```

Comment:
I must have made a mistake here. It's indeed not necessary to prevent a warning.